public/pages/Overview/containers/Overview/Overview.tsx

```diff
@@ -101,15 +103,17 @@ export const Overview: React.FC<OverviewProps> = (props) => {
     () => new OverviewViewModelActor(saContext?.services, context?.notifications!),
     [saContext?.services, context]
   );
-
+  const currentNavGroup = useObservable(getChrome().navGroup.getCurrentNavGroup$());
+  const isSecurityAnalyticsUesCase = currentNavGroup?.id === SECURITY_ANALYTICS_USE_CASE_ID;
```

Comment:
Nit: typo

 isSecurityAnalytics**Ues**Case

Reply:
thanks for catching this, updated.